src/components/SwipeButton/index.js

```diff
@@ -177,6 +179,7 @@ SwipeButton.defaultProps = {
   thumbIconBorderColor: THUMB_ICON_BORDER_COLOR,
   thumbIconStyles: {},
   title: 'Swipe to submit',
+  titleMaxFontScale: 2,
```

Comment:
I think we should skip default value config for this to avoid effecting usage in all existing projects. I do understand that `titleMaxFontScale: 2` gives out of the box support for large font settings but this would be a breaking change.

Reply:
i'm not sure i need to test it

Reply:
BTW there is no UT in your component, how can you be sure that any contributor will break existing features, this is a thing that you should add i think.

Reply:
Yes, test cases are really important, totally agree with you. But I'm not finding time at all. When I find little bit time, I use it to improve/support important requirements logged by users. I had created a [ticket](https://github.com/UdaySravanK/RNSwipeButton/projects/1#card-44053861) in project, contributions will be appreciated.